integration/helpers/deployer/exceptions/exceptions.py

```diff
@@ -15,10 +15,9 @@ def __init__(self, message: str) -> None:
 
 
 class ChangeEmptyError(UserException):
-    def __init__(self, stack_name):
+    def __init__(self, stack_name: str) -> None:
         message_fmt = "No changes to deploy. Stack {stack_name} is up to date"
-        super().__init__(message=message_fmt.format(stack_name=self.stack_name))
-        self.stack_name = stack_name
```

Comment:
Is this not needed?

Reply:
I check the usage of `ChangeEmptyError`, its `stack_name` is not used elsewhere